core/src/test/scala/org/apache/spark/scheduler/WorkerDecommissionSuite.scala

```diff
@@ -83,4 +85,29 @@ class WorkerDecommissionSuite extends SparkFunSuite with LocalSparkContext {
     val asyncCountResult = ThreadUtils.awaitResult(asyncCount, 20.seconds)
     assert(asyncCountResult === 10)
   }
+
+  test("Verify executors are decommissioned in batches") {
```

Comment:
Could you add `SPARK-35627: ` as the test name prefix?

Reply:
Done. Thanks.